src/Renderer/ThreeExtended/OBB.js

```diff
@@ -7,17 +7,21 @@
 
 import * as THREE from 'three';
 
-function OBB(min, max, lookAt, translate) {
+function OBB(min, max, lookAt, translate, up) {
     THREE.Object3D.call(this);
     this.box3D = new THREE.Box3(min, max);
 
     this.natBox = this.box3D.clone();
 
     this.quaInv = this.quaternion.clone().inverse();
 
+    if (up) {
+        this.up.set(up.x, up.y, up.z);
```

Comment:
yes, I know
but it's possible to instantiate the box by swapping the min and max and translations in `builderEllipsoidTile`.
It would also be necessary to have the BBOXhelper to verify